.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,29 @@
+name: scheduled
+on:
+  schedule:
```

Comment:
Is it not possible to add the cron schedule to the main CI job description? I've done this in other projects and I believe it used to work.

Reply:
Ah, I guess the problem is that scheduled runs only work on the `main` branch not release branches?

> Scheduled workflows will only run on the default branch.

In that case I think I would prefer including the scheduled job for the main branch in `ci.yml` and explicitly state this workflow is to run on old release branches.

Reply:
Yes, this is why it's triggered indirectly. I guess we're mostly ok with main since new PRs pop out regularly.

Most problems we have are with the release branches -- like, you're about to cut a .z release and find out that nothing is working, and you need to backport 27 commits (the number is not entirely unrealistic) just for the CI  to work again.

Reply:
It would be nice to have a comment explaining the need for this, since my first thought was why not have the `scheduled` section added to `ci.yml`.

Reply:
Added a comment.